Makefile

```diff
@@ -52,19 +50,15 @@ $(FIRSTGOPATH)/bin/golangci-lint:
 test: $(packr) $(gosources) go.mod go.sum
 	go test -tags=integration ./...
 
-docker-image: vendor $(packr) $(gosources) Dockerfile .dockerignore
+docker-image: $(packr) $(gosources) Dockerfile .dockerignore
 	docker build -t quorumcontrol/tupelo:$(TAG) .
 
-$(FIRSTGOPATH)/bin/modvendor:
-	GO111MODULE=off go get -u github.com/goware/modvendor
-
-install: $(packr) $(gosources) go.mod go.sum
-	go install -a -gcflags=-trimpath=$(CURDIR) -asmflags=-trimpath=$(CURDIR)
+install: tupelo
+	cp tupelo $(FIRSTGOPATH)/bin/
```

Comment:
This can probably wait for another pr since it might have ripple effects, but I think it might be a good idea to install this under a more general prefix as well. It might make it easier for a new signer to just install tupelo using a standard package manager.

Reply:
Yeah i'll leave this for another PR --- there are a lot of improvements I was noticing that we can make here in future PRs